sys/sys/sysent.h

```diff
@@ -49,7 +49,7 @@ typedef	int	sy_call_t(struct thread *, void *);
 
 typedef	void	(*systrace_probe_func_t)(struct syscall_args *,
 		    enum systrace_probe_t, int);
-typedef	void	(*systrace_args_func_t)(int, void *, uint64_t *, int *);
+typedef	void	(*systrace_args_func_t)(int, void *, uintcap_t *, int *);
```

Comment:
Won't this break 32-bit? Probably wants to be kuint64cap_t (or some new equivalent uint64cap_t)?

Reply:
I don't think so. syscall arguments are always register-sized; they are passed in as syscallarg_t, aka register_t. And, when they reach dtrace_probe(), they are cast to uintptr_t. So, the previous use here of uint64_t seems incorrect.